.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 # black complains loudly that fio-histo-log-pctile.py is not a python file
```

Comment:
Funny that the comment justifies one of three entries here, but then it used to be one of four. 😁

Reply:
As @webbnh says: "Every day, in every way, I'm getting better and better...." [1]  - eventually there will only be one entry and the comment will be correct :-)

[1] Of course, he was quoting Inspector Clouseau, so maybe... Never mind!

Reply:
Ha. Meaning, we'll never be done. Job security.

Reply:
> eventually there will only be one entry and the comment will be correct :-)

I just put up PR #2972, which, once the conflict with this PR is resolved, will have that very effect...

> he was quoting Inspector Clouseau

Actually, that was Commissioner Dreyfus, Clouseau's boss....